starknet_py/tests/e2e/docs/guide/test_serializing.py

```diff
@@ -0,0 +1,45 @@
+# pylint: disable=import-outside-toplevel
```

Comment:
The guide showing how higher level api like Contract uses serialization was removed. Wouldn't having an example like this (in the separate snipped) be useful?

Reply:
using_existing_contracts covers that